### DISCUSSION TITLE
多字段且部分字段是json字符串如何使用文件导入

### DISCUSSION THREAD

**Discussion by pang-pang-ya** [apache/doris#20913] [A - General / Q&A]
csv文件导入，里面的内容包含json格式的内容[字段id,name,json_data]，eg:6,name,{a:1,b:2}，如何使用java进行文件导入：
id,name,json_data
6,name,{a:1,b:2}
.......................

**Comment by pang-pang-ya** [comment]
这种数据使用逗号分割之后，列数不一致，就会报错。

**Reply by yagagagaga** [reply]
You can write a program that processes this data into a streamload recognizable format.
